index.html

```diff
@@ -611,10 +738,10 @@ <h4>Script Type</h4>
                   and MAY also contain <a>Original</a> <a>Text</a> objects from the <a>Original Language Transcript</a>
                   or <a>Translation</a> <a>Text</a> objects in other languages for context and quality verification.
                   They MAY also contain links to audio and mixing instructions for the purpose of producing an audio track incorporating the recordings.</p>
-                <aside class="note">Even though the output of the dubbing workflow is in the same language as those
-                <a>Text</a> objects with the matching language,
-                the <a>Text Language Source</a> of those Text objects remains <a>Translation</a>
-                since they differ from the original language of the dialogue.</aside>
+                <aside class="note"><a>As-recorded Scripts</a> are not considered to be transcripts
+                  even though they are adjusted to match the recorded output of the dubbing workflow.
+                  The <a>Translation</a> <a>Text</a> objects within them retain their <a>Text Language Source</a>,
+                  so that the source language from which they were translated remains available.</aside>
```

Comment:
Something like:

```suggestion
                <aside class="note">The <a>Translation</a> <a>Text</a> objects within <a>As-recorded Scripts</a> retain their <a>Text Language Source</a>,
                  so that the source language from which they were translated remains available.</aside>
```

?

Reply:
Is it missing "can" in "can retain"?

Reply:
I think Translation objects must have a computed value of `daptm:langSrc` other than the empty string. `can` would make it a `MAY` constraint.

Reply:
Agree with @andreastai we should not make this appear to be optional.

Reply:
I was tempted to separate this into two notes, one for each point, but on close reading I don't think it matters whether As-Recorded scripts are transcripts or scripts - nothing in the spec seems to depend on this distinction. So I'll delete that part.

Reply:
Done in 10b1a0e